File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/ServerIpmiLiveRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Request body to setup an IPMI Live connection.
  /// </summary>
  [DataContract]
  public class ServerIpmiLiveRequest {
    /// <summary>
    /// Asset ID
    /// </summary>
    /// <value>Asset ID</value>
    [DataMember(Name="asset", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "asset")]
    public int? Asset { get; set; }

    /// <summary>
    /// Your IP Address you wish to connect to the IPMI system from.
    /// </summary>
    /// <value>Your IP Address you wish to connect to the IPMI system from.</value>
    [DataMember(Name="ip", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "ip")]
    public string Ip { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ServerIpmiLiveRequest {\n");
      sb.Append("  Asset: ").Append(Asset).Append("\n");
      sb.Append("  Ip: ").Append(Ip).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/server_ipmi_live_request.dart
part of swagger.api;

class ServerIpmiLiveRequest {
  /* Asset ID */
  int asset = null;
/* Your IP Address you wish to connect to the IPMI system from. */
  String ip = null;

  ServerIpmiLiveRequest();

  @override
  String toString() {
    return 'ServerIpmiLiveRequest[asset=$asset, ip=$ip, ]';
  }

  ServerIpmiLiveRequest.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    asset = json['asset'];
    ip = json['ip'];
  }

  Map<String, dynamic> toJson() {
    return {
      'asset': asset,
      'ip': ip
     };
  }

  static List<ServerIpmiLiveRequest> listFromJson(List<dynamic> json) {
    return json == null ? new List<ServerIpmiLiveRequest>() : json.map((value) => new ServerIpmiLiveRequest.fromJson(value)).toList();
  }

  static Map<String, ServerIpmiLiveRequest> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ServerIpmiLiveRequest>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ServerIpmiLiveRequest.fromJson(value));
    }
    return map;
  }
}
